app/src/main/java/com/dimagi/biometric/activities/BaseActivity.java

```diff
@@ -71,6 +71,13 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     protected void loadFragment(Fragment fragment) {
+        ArrayList<String> errors = validateRequiredParams();
+        if (errors.size() > 0) {
+            Bundle args = new Bundle();
+            String errorStr = createErrorStr(errors);
+            args.putString("errors", errorStr);
+            fragment.setArguments(args);
+        }
```

Comment:
Can we refactor this to a separate method?

Reply:
[71a915d](https://github.com/dimagi/biometric-android/pull/3/commits/71a915d5adbdedb8a00e085a5796b687e9903e21)